EMBEDDED_PROTOCOL_CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.4.0
```

Comment:
We now need a rebase and another version bump here.

Reply:
Done in https://github.com/sass/sass/pull/3728/commits/1c36ec5577d75558f6f18e95c9f77358167b826b

Reply:
Please also update `spec/EMBEDDED_PROTOCOL_VERSION`. Thanks.

Reply:
Apologies, done in https://github.com/sass/sass/pull/3728/commits/7ec5cb7b87a9e3b1eb495935953fc73ec0d9156f